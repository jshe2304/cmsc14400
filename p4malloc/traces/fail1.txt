10000
4
8
1
a 0 10000
a 1 2000
a 2 2000
a 3 2000
f 0
f 1
f 2
f 3