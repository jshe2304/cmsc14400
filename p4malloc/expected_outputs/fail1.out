1	24	0
1	10048	0
0	-1	-1
0	-1	-1
